### ISSUE TITLE
dropdown control and dropdown items need accessibility improvements

### ISSUE THREAD

**Issue by FennaKort** [FennaKort/php-and-accessible-menu-learning#3]
[] dialogue for dropdown must be aria-labelled
	[] dialogue items aren't showing up in screen-reader accessible menu
	[] dialogue needs to have aria-expanded be changed when dropdown is opened

**Comment by FennaKort** [comment]
# [Inclusive Components: Menus and menu buttons](https://inclusive-components.design/menus-menu-buttons/)
If the contents of a dropdown or offscreen nav are revealed on hover or on focus, that should be a change in visual styling, and should not/does not actually change the state of the document for assistive technology users. I'm assuming this comment would relate to accordions that hide/reveal on hover/focus as well. 

Recommends using a table of contents on a dedicated page instead of a dropdown subnav. Ie., no dropdowns in nav menu; all nav links go to specific pages. A link that would have been the top level link or label of a dropdown submenu instead goes to a page that displays that same information as a table of contents. I'm not sure I totally agree that this is always preferable, because then you're not taking advantage of the nested table of contents navigation structure, but this could make more sense to use in cases where there is a lot of information to get through.
